src/room.py

```diff
@@ -1,2 +1,27 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes. 
+
+class Room():
+    def __init__(self, name, description, items=[], n_to = None, s_to = None, e_to = None, w_to = None):
```

Comment:
No need to pass in the n_to,s_to,e_to,w_to parameters in the __init__ function  since we are already defining it in adv.py